src/field_weather_effect.c

```diff
@@ -150,11 +153,14 @@ bool8 Clouds_Finish(void)
     return FALSE;
 }
 
+STATIC_ASSERT(OW_SHADOW_INTENSITY >= 0 && OW_SHADOW_INTENSITY <= 16, ObjEventShadowTransparencyNotInRange)
```

Comment:
This static assert should not be in this file imo, it's wholly unrelated to weather

Reply:
`BASE_SHADOW_INTENSITY` is only used here and in `src/field_weather.c` though, with a weather-related function (`Weather_SetBlendCoeffs`)